### PR TITLE
Linux fragments

### PR DESCRIPTION
**Pull request by angolini**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

There are two different sections regarding "linux fragments," but only one of them shows the technical instructions on how to configure the kernel, so add the cross-reference.

Another important change is on the title, as for a lot of readers, I would expect they don't know what the term "Linux fragment" means, so I suggest another title.

While on that, fix some other minor stuff.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e97b2648f921068a0076c218f3293452af8a4d7e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1945/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-kernel.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. _ref-linux-kernel:
 
-Linux Kernel
-============
+Linux® Kernel
```
Comment (kprosise) [inline source/reference-manual/linux/linux-kernel.rst:3]:
```suggestion
Linux Kernel
```
Trademark doesn't go in the header/section title, but in the body of the text.

File: source/reference-manual/linux/linux-kernel.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+=============
 
 A common and unified Linux Kernel source tree is provided and used by
```
Comment (kprosise) [inline source/reference-manual/linux/linux-kernel.rst:6]:
```suggestion
A common and unified Linux® Kernel source tree is provided and used by
```

File: source/reference-manual/linux/linux-kernel.rst [hunk 3]
```diff
@@ -29,9 +29,19 @@ repository, so the same development workflow and documentation applies.
 See the `Yocto Project Linux Kernel Development Manual`_ for more information
 on how to work and manage the kernel metadata and configuration fragments.
 
+The Porting Guide includes the section :ref:`ref-pg-how-to-configure-linux` on
+on how to add a custom Linux Kernel configuration which can be used to add:
+
+* the complete machine configuration.
+
+* to add fragments: a set of ``CONFIG_`` variables working to change
+  a default machine configuration.
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-kernel.rst:38]: `to add` is already in line 33
```suggestion
* configuration fragments: a set of ``CONFIG_`` variables working to change
  a default machine configuration.
```

File: source/reference-manual/linux/linux-kernel.rst [hunk 4]
```diff
@@ -29,9 +29,19 @@ repository, so the same development workflow and documentation applies.
...
+
+
+
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-kernel.rst:41]: can these lines be removed?
```suggestion
```

**Comment by doanac** [comment]
Docs for b914bc98f2622533bb27ed069a863041ffa116f1 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1948/docs/artifacts/html/index.html

**Comment by angolini** [comment]
@kprosise @vanmaegima review accepted

**Review by kprosise — APPROVED** [review]
lgtm

**Pull request merged**